.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+branches:
+  only:
+    - master
+    - /\bv[0-9]/
+dist: xenial
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+install:
+  - pip install -e .[dev]
```

Comment:
what does that `[dev]` ?

Reply:
something we don't need for now (it installs tox), so I'll remove it